adapters/beachfront/beachfront.go

```diff
@@ -24,41 +22,45 @@ const VideoEndpoint = "https://reachms.bfmio.com/bid.json?exchange_id="
 const VideoEndpointSuffix = "&prebidserver"
 
 const beachfrontAdapterName = "BF_PREBID_S2S"
-const beachfrontAdapterVersion = "0.3.0"
+const beachfrontAdapterVersion = "0.6.0"
+
+const DefaultVideoWidth = 300
+const DefaultVideoHeight = 250
 
 type BeachfrontAdapter struct {
 }
 
-type BeachfrontRequests struct {
-	Banner BeachfrontBannerRequest
-	Video  BeachfrontVideoRequest
+type beachfrontRequests struct {
+	Banner beachfrontBannerRequest
+	Video  []beachfrontVideoRequest
+	Audio  openrtb.Audio
```

Comment:
Out of curiosity, why is `Audio` and `Native` added here when your adapter only supports banner and video ads?

Reply:
Unfounded optimism.

Reply:
I was messing around with them a bit and didn't catch that they were still in the enum declaration.